src/rules/prefer-lowercase-title.ts

```diff
@@ -70,11 +68,14 @@ export default createRule<
           ignore: {
             type: 'array',
             items: {
+              type: 'string',
+              // for some reason TypeScript thinks this _must_ be a read-only
```

Comment:
We probably need to update our parameter types to be `const` parameters to avoid weirdness like this.